docs/Modding/musicsets.rst

```diff
@@ -0,0 +1,106 @@
+Musicsets Overview
+******************
+
+A Musicset is a soundtrack for Freeciv21 and is often documented in a single :literal:`.musicspec` file with
+references to the media files in a separate sub-directory.
+
+The soundtrack is broken up into time periods following the technology tree and is split by two moods: peace
+and combat. Peace is generally defined when you are "farming" and growing your civilization with no fighting
+going on at all. Any kind of fighting, inlcuding defense of your cities will trigger combat and the music mood
+will change. After 16 turns of no combat the music will return to peace.
+
+At the beginning of the game, you select the style of music you want to hear by picking the city style of your
+civilization. The current choices are:
+
+* European
+* Classical
+* Tropical
+* Asian
+* Babylonian
+* Celtic
+
+The city style is automatically selected for you when you pick your nation, but you can change it if you like.
+
+The discovery of :strong:`University` changes the music for all city styles for the Renaissance Age. All further
+disoveries noted below will change the music for all city styles/nations.
+
+The discovery of :strong:`Railroad` brings in the Industrial Age.
+
+The discovery of :strong:`Automobile` brings in the Electric Age.
+
+The discovery of :strong:`Rocketry brings` in the Modern Age.
+
+The discovery of :strong:`Superconductors` brings in the Post-Modern Age soundtrack. This is the final
+soundtrack of the game.
```

Comment:
I would document that this is for the default rulesets and linked to the visual appearance of the cities.